# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1308068440 14400
#      Tue Jun 14 12:20:40 2011 -0400
# Branch stable
# Node ID 8d2d7f9a17be9ec5c9d89780f781ba713142fe2c
# Parent  0cb4e502fe5e858fe979b3bfa0072f31e40a7c49
maint: update NEWS

* NEWS: Add info about changes to ChangeLog files.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -38,16 +38,22 @@ Summary of important user-visible change
  ** The get_forge_pkg function is now private.
 
  ** The rectangle_lw, rectangle_sw, triangle_lw, and triangle_sw
     functions are now private.
 
  ** The logistic_regression_derivatives and logistic_regression_likelihood
     functions are now private.
 
+ ** ChangeLog files in the Octave sources are no longer maintained
+    by hand.  Instead, there is a single ChangeLog file generated from
+    the Mercurial version control commit messages.  Older ChangeLog
+    information can be found in the etc/OLD-ChangeLogs directory in the
+    source distribution.
+
 Summary of important user-visible changes for version 3.4:
 ---------------------------------------------------------
 
  ** BLAS and LAPACK libraries are now required to build Octave.  The
     subset of the reference BLAS and LAPACK libraries has been removed
     from the Octave sources.
   
  ** The ARPACK library is now distributed with Octave so it no longer
